doc/api/child_process.md

```diff
@@ -339,6 +339,8 @@ changes:
     When this option is provided, it overrides `silent`. If the array variant
     is used, it must contain exactly one item with value `'ipc'` or an error
     will be thrown. For instance `[0, 1, 2, 'ipc']`.
+  * `windowsVerbatimArguments` {Boolean} No quoting or escaping of arguments is
```

Comment:
A nit: primitive types are lowercased, so this should be `{boolean}` here and in other additions,

Reply:
fixed.